src/frontend/test_runner/tests/testdata/appendonly.yaml

```diff
@@ -0,0 +1,22 @@
+# This file is automatically generated. See `src/frontend/test_runner/README.md` for more information.
+- sql: |
+    create table t1 (v1 int, v2 int) with ('appendonly' = true);
+    select v1, max(v2) as mx2 from t1 group by v1;
+  stream_plan: |
+    StreamMaterialize { columns: [v1, agg#0(hidden), mx2], pk_columns: [v1], appendonly: true }
```

Comment:
This derivation is wrong. HashAgg will never produce append-only result. We can fix this later.

Reply:
Oh. you are right. The output of hash agg may produce update (U-, U+)